contracts/TestHelpers/ChallengesStoreMock.sol

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright: Ambrosus Technologies GmbH
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+
+pragma solidity ^0.4.23;
+
+import "../Storage/ChallengesStore.sol";
+
+
+contract ChallengesStoreMock is ChallengesStore {
+
+    constructor(Head _head) public ChallengesStore(_head) {
+    }
+
+    function injectChallenge(
+        address sheltererId,
+        bytes32 bundleId,
+        address challengerId,
+        uint feePerChallenge,
+        uint64 creationTime,
+        uint8 activeCount,
+        uint sequenceNumber)
+    public payable onlyContextInternalCalls returns (bytes32)
```

Comment:
IMO: onlyContextInternal here is somewhat redundant, as this is only for tests. But ok:D